projects/media-viewer/src/lib/annotations/annotation-set/annotation-set.component.spec.ts

```diff
@@ -298,7 +298,7 @@ describe('AnnotationSetComponent', () => {
     })
   );
 
-  it('should create text highlight',
+  xit('should create text highlight', // TODO this keeps randomly failing on travis
```

Comment:
is this working locally? is it sharing an object with another test?

Reply:
yeah it does work locally fine.. think gonna revert this as they do random fail just not this one

Reply:
ok